subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -597,6 +585,15 @@ static void isr_rx(void *param)
 				lll->payload[bis_idx][payload_tail] = NULL;
 
 				iso_rx_put(node_rx->hdr.link, node_rx);
+			} else {
+				node_rx = ull_iso_pdu_rx_alloc_peek(3U);
+				if (node_rx) {
+					ull_iso_pdu_rx_alloc();
+					isr_rx_iso_data_invalid(lll, bis_idx,
+								bn, node_rx);
+
+					iso_rx_put(node_rx->hdr.link, node_rx);
+				}
```

Comment:
Can you elaborate on why we need to peek 3 RX here? And preferably also add that as a comment to the code?

Reply:
Added comments in code.